src/gc/Makefile

```diff
@@ -0,0 +1,59 @@
+SRCDIR := $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
+JULIAHOME := $(abspath $(SRCDIR)/../..)
+JULIASRC := $(abspath $(JULIAHOME)/src)
+JULIASUPPORT := $(abspath $(JULIAHOME)/src/support)
+BUILDDIR := .
+# Get the name of the parent of BUILDDIR since there will be a few automatically generated files there
+SRCBUILDDIR := $(abspath $(BUILDDIR)/..)
+
+include $(JULIAHOME)/Make.inc
+
+JCFLAGS += $(CFLAGS)
+JCXXFLAGS += $(CXXFLAGS)
+JCPPFLAGS += $(CPPFLAGS)
+JLDFLAGS += $(LDFLAGS)
+
+SRCS := gc-debug gc-page-profiler gc-pages gc-stacks gc
+
+OBJS := $(SRCS:%=$(BUILDDIR)/%.o)
+DOBJS := $(SRCS:%=$(BUILDDIR)/%.dbg.obj)
+
+HEADERS := $(wildcard *.h) $(LIBUV_INC)/uv.h
+
+FLAGS := $(HFILEDIRS:%=-I%) -I$(LIBUV_INC) -I$(UTF8PROC_INC) -I$(JULIASRC) -I$(JULIASUPPORT) -I$(SRCBUILDDIR) -I. -DJL_LIBRARY_EXPORTS_INTERNAL
+FLAGS += -Wall -Wno-strict-aliasing -fvisibility=hidden -Wpointer-arith -Wundef -fPIC
```

Comment:
So these flags are slightly different from the ones in `src`, maybe we should move them to a variable in Make.inc like runtimeflags or something like that. Just so they are kept in sync

Reply:
These flags are fairly close to the ones used by libsupport.

In libsupport they are not defined in the Make.unc, but rather are defined in the corresponding Makefile.

I just followed the same in here.

Reply:
These flags are fairly close to the ones used by libsupport.

In libsupport they are not defined in the Make.inc, but rather are defined in the corresponding Makefile.

I just followed the same in here.